arxiv/marxdown/tests/test_serve.py

```diff
@@ -202,3 +206,16 @@ def test_serve_with_htm(self):
                              status.HTTP_302_FOUND)
             self.assertEqual(response.headers['Location'],
                              'http://localhost/baz/redirectme')
+
+    @mock.patch(f'{factory.__name__}.config', CONFIG)
+    def test_serve_static(self):
+        """Requests for static URLs should be redirected."""
+        app = factory.create_web_app()
+        client = app.test_client()
+
+        with app.app_context():
+            response = client.get('/notapage.txt', follow_redirects=False)
+            u = "https://%s.s3.amazonaws.com/static/arxiv.marxdown/%s/%s/%s" \
```

Comment:
Minor: it might be helpful to have another test case where `FLASKS3_CDN_DOMAIN` is defined.